src/chess/chess_game.rs

```diff
@@ -180,6 +183,18 @@ fn chess_color_to_game_color(c: chess::Color) -> GameColor {
     }
 }
 
+impl PartialEq for ChessPosition {
+    fn eq(&self, other: &Self) -> bool {
+        self.board == other.board

```

Comment:
couldn't find your original comment, but we should not included here the 50 rule count, as we want to check for 'equal' positions, and we expect two positions to be equal if the board is the same, even if the 50 rule counter is different.

Reply:
haha yes i deleted it